editor/translations/af.po

```diff
@@ -3666,11 +3666,6 @@ msgstr "Gunstelinge:"
 msgid "Status: Import of file failed. Please fix file and reimport manually."
 msgstr ""
 
-#: editor/filesystem_dock.cpp
```

Comment:
@HEAVYPOLY seems like you've lost the changes that's actually needed.

Reply:
Sorry about that...I think I've got it this time!